x-pack/plugins/security_solution/public/common/components/integrations_popover/helpers.tsx

```diff
@@ -10,29 +10,58 @@ import { capitalize } from 'lodash';
 import React from 'react';
```

Comment:
Maybe we should move `integrations_popover` to `x-pack/plugins/security_solution/public/detections/components/rules/related_integrations` folder, since I don't think it's a common component - it's only used from 2 rule management pages so far, so I'd keep it as close to those pages as possible.

Reply:
Addressed in https://github.com/elastic/kibana/pull/133050 👍